tools/spell_checker/dictionary.txt

```diff
@@ -849,7 +849,8 @@ diverges
 divil
 dizzied
 dodecahedron
-doesnt
+doesn
+doesn't
```

Comment:
dictionary doesn't handle apostrophes at all, it will read it as words `doesn`, `t`

Reply:
Should I add `doesn` and `isn` then or just ignore the dictionary and remove my changes there?

Reply:
remove and ignore it, someone need to fix it in another, proper way